pilot/cmd/pilot-agent/main.go

```diff
@@ -144,6 +144,8 @@ var (
 	useTokenForCSREnv   = env.RegisterBoolVar("USE_TOKEN_FOR_CSR", false, "CSR requires a token").Get()
 	credFetcherTypeEnv  = env.RegisterStringVar("CREDENTIAL_FETCHER_TYPE", "",
 		"The type of the credential fetcher. Currently supported types include GoogleComputeEngine").Get()
+	parseTokenEnv = env.RegisterBoolVar("PARSE_TOKEN", false,
```

Comment:
When is this ever set to true? seems currently never?

Reply:
It's set by the environment variable.